### ISSUE TITLE
Crashing - UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte

### ISSUE THREAD

**Issue by typewriter1** [Epihaius/panda3dstudio#4]
I am getting a crash whenever I:

- Attempt to undo using ctrl-z or the back button
- Move a vertex: I can drag it but when I release the mouse to drop it, it crashes

Here is the full traceback:

```
C:\Users\avise\Desktop\Games\tools\panda3dstudio-master>ppython main.py
Known pipe types:
  wglGraphicsPipe
(all display modules loaded.)
Traceback (most recent call last):
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\__init__.py", line 20, in handle_pending_tasks
    PendingTasks.handle(["object", "ui"], True)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\base.py", line 356, in handle
    task()
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\base.py", line 175, in __call__
    self._func()
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\obj_mgr.py", line 334, in <lambda>
    get_task = lambda objs: lambda: self.__add_to_history(objs, prop_id, value)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\obj_mgr.py", line 307, in __add_to_history
    obj_data[obj.get_id()] = obj.get_data_to_store("prop_change", prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\model.py", line 234, in get_data_to_store
    data.update(self._geom_obj.get_data_to_store(event_type, prop_id))
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\prim\cone.py", line 604, in get_data_to_store
    "prop_change", "all"))
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\data\history.py", line 192, in get_property_to_store
    prev_time_ids = Mgr.do("load_last_from_history", obj_id, unique_prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\mgr.py", line 134, in do
    return task_handler(*args, **kwargs)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\history.py", line 429, in __load_last_value
    value = self.__load_property_value(hist_file, last_time_id, obj_id, prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\history.py", line 663, in __load_property_value
    prop_val_pickled = hist_file.read_subfile(subfile_index)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
:task(error): Exception occurred in PythonTask handle_pending_tasks
Traceback (most recent call last):
  File "main.py", line 44, in <module>
    App()
  File "main.py", line 41, in __init__
    mgr.get_base().run()
  File "C:\Panda3D-1.10.0-x64\direct\showbase\ShowBase.py", line 3040, in run
    self.taskMgr.run()
  File "C:\Panda3D-1.10.0-x64\direct\task\Task.py", line 531, in run
    self.step()
  File "C:\Panda3D-1.10.0-x64\direct\task\Task.py", line 485, in step
    self.mgr.poll()
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\__init__.py", line 20, in handle_pending_tasks
    PendingTasks.handle(["object", "ui"], True)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\base.py", line 356, in handle
    task()
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\base.py", line 175, in __call__
    self._func()
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\obj_mgr.py", line 334, in <lambda>
    get_task = lambda objs: lambda: self.__add_to_history(objs, prop_id, value)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\obj_mgr.py", line 307, in __add_to_history
    obj_data[obj.get_id()] = obj.get_data_to_store("prop_change", prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\model.py", line 234, in get_data_to_store
    data.update(self._geom_obj.get_data_to_store(event_type, prop_id))
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\prim\cone.py", line 604, in get_data_to_store
    "prop_change", "all"))
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\geom\data\history.py", line 192, in get_property_to_store
    prev_time_ids = Mgr.do("load_last_from_history", obj_id, unique_prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\base\mgr.py", line 134, in do
    return task_handler(*args, **kwargs)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\history.py", line 429, in __load_last_value
    value = self.__load_property_value(hist_file, last_time_id, obj_id, prop_id)
  File "C:\Users\avise\Desktop\Games\tools\panda3dstudio-master\src\core\history.py", line 663, in __load_property_value
    prop_val_pickled = hist_file.read_subfile(subfile_index)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
```

**Comment by rdb** [comment]
This should have already been fixed in Panda3D (see panda3d/panda3d#290).  Could you make sure you have a recent enough development build?

**Comment by typewriter1** [comment]
I updated to the latest build and that fixed it!
